data-raw/process_data_sets_meredith.R

```diff
@@ -0,0 +1,162 @@
+library(tidyverse)
+library(stringr)
+library(lubridate)
+library(janitor)
+
+# ahca_polls ----------------------------------------------------------------------
+ahca_polls <- read_csv("data-raw/ahca-polls/ahca_polls.csv") %>% 
+  clean_names() %>% 
+  mutate(
+    start = as.Date(start, "%m/%d/%y"),
+    end = as.Date(end, "%m/%d/%y"),
+    pollster = as.factor(pollster)
+  )
+devtools::use_data(ahca_polls, overwrite = TRUE)
+
+
+# bachelorette ---------------------------------------------------------------------
+bachelorette <- read_csv("data-raw/bachelorette/bachelorette.csv") %>%
+  clean_names() %>%
+  mutate(
+    season = as.factor(season),
+    elimination_1 = as.factor(elimination_1),
+    elimination_2 = as.factor(elimination_2),
+    elimination_3 = as.factor(elimination_3),
+    elimination_4 = as.factor(elimination_4),
+    elimination_5 = as.factor(elimination_5),
+    elimination_6 = as.factor(elimination_6),
+    elimination_7 = as.factor(elimination_7),
+    elimination_8 = as.factor(elimination_8),
+    elimination_9 = as.factor(elimination_9),
+    elimination_10 = as.factor(elimination_10),
+    dates_1 = as.factor(dates_1),
+    dates_2 = as.factor(dates_2),
+    dates_3 = as.factor(dates_3),
+    dates_4 = as.factor(dates_4),
+    dates_5 = as.factor(dates_5),
+    dates_6 = as.factor(dates_6),
+    dates_7 = as.factor(dates_7),
+    dates_8 = as.factor(dates_8),
+    dates_9 = as.factor(dates_9),
+    dates_10 = as.factor(dates_10)
+  ) %>%
+  filter(
+    season != "SEASON"
+  )
+devtools::use_data(bachelorette, overwrite = TRUE)
+
+
+# candy-power-rankings -------------------------------------------------------------
+candy_rankings <- read_csv("data-raw/candy-power-ranking/candy-data.csv") %>%
+  clean_names() %>%
+  mutate(
+    chocolate = as.logical(chocolate),
+    fruity = as.logical(fruity),
+    caramel = as.logical(caramel),
+    peanutyalmondy = as.logical(peanutyalmondy),
+    nougat = as.logical(nougat),
+    crispedricewafer = as.logical(crispedricewafer),
+    hard = as.logical(hard),
+    bar = as.logical(bar),
+    pluribus = as.logical(pluribus)
+  )
+devtools::use_data(candy_rankings, overwrite = TRUE)
```

Comment:
Some of the apostrophes in `competitorname` came out weird in the conversion. Ex: `HersheyÕs Kisses` and `WelchÕs Fruit Snacks`.

Whenever working with data, the *most* critical step, above beyond model fitting, any kind of EDA or data cleaning, is just looking at the values, using `View(dataframe)` or `glimpse(dataframe)`. People, even people with PhD's, often ignore this step.